app-ios/Modules/Sources/Navigation/RootViewModel.swift

```diff
@@ -0,0 +1,26 @@
+import Dependencies
+import Foundation
+import KMPContainer
+import Model
+
+struct RootViewState: ViewModelState {
+    var isStampEnabled: LoadingState<Bool> = .initial
```

Comment:
[nits]
Since we have this ticket https://github.com/DroidKaigi/conference-app-2023/issues/989, it might be better to use the word "achievement" instead.